src/win/util.c

```diff
@@ -484,7 +484,7 @@ uint64_t uv__hrtime(double scale) {
    * performance counter interval, integer math could cause this computation
    * to overflow. Therefore we resort to floating point math.
    */
-  return (uint64_t) ((double) counter.QuadPart * hrtime_interval_ * scale);
+  return (uint64_t) (counter.QuadPart * (uint64_t) scale / hrtime_interval_);
```

Comment:
@Arnavion is correct, only it's 2^53.  Larger values get approximated with increasingly higher error bars:

```
> Math.pow(2,53) === Math.pow(2,53)-1
false
> Math.pow(2,53) === Math.pow(2,53)+1
true
```